lib/routes/account_required_actions.dart

```diff
@@ -44,60 +39,57 @@ class AccountRequiredActionsIndicator extends StatefulWidget {
 
 class AccountRequiredActionsIndicatorState
     extends State<AccountRequiredActionsIndicator> {
-  StreamSubscription<void> _promptEnableSubscription;
-  StreamSubscription<BackupSettings> _settingsSubscription;
-  BackupSettings _currentSettings;
+  StreamSubscription<dynamic> _promptBackupSubscription;
   bool showingBackupDialog = false;
-  bool _init = false;
 
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
-    if (!_init) {
```

Comment:
why was this removed? `if (!_init)`
As far as I understand it protects this logic from running more than once.

Reply:
Using an init flag inside the change dependencies method is an error; It lead to lifecycle bugs (I had solved several bugs like that on previous PRs)

If there is something we want to do exactly one time, we shall use `initState` 

Honestly, I don't see the need for but I can move this code to init state if you prefer @roeierez

Reply:
I think it would be best to use initState as I think right now we also have a bug since the `didChangeDependencies` can be invoked several times.

Reply:
Ok, I'll move to initState; There is no bug right now because I'm cleaning the subscription properly; the only side effect is we are wasting some cpu time here.